checks/collector.py

```diff
@@ -222,10 +240,8 @@ def __init__(self, agentConfig, emitters, systemStats, hostname):
             except Exception:
                 log.exception('Unable to load custom check module %s' % module_spec)
 
-        # Resource Checks
-        self._resources_checks = [
-            ResProcesses(log, self.agentConfig)
-        ]
+        # Resource Checks (deprecated)
+        self._resources_checks = []
```

Comment:
I can see you're keeping this in for legacy reasons, but I want to double check, as we now have a relatively large if branch that would never be executed [here](https://github.com/DataDog/dd-agent/blob/olivielpeau/use-go-process-map/checks/collector.py#L380-L395).